pkg/synchronizer/kubernetes/sync_server.go

```diff
@@ -56,6 +56,8 @@ type ResourceMap struct {
 // KubeSynchronizer handles resources to a kube endpoint
 type KubeSynchronizer struct {
 	Interval      int
+	lCachedClt    *cachedClient
```

Comment:
Sorry for being picky but I would just spell it out like `localCachedClient` and `remoteCachedClient`. I understand these but any contributer who will maintain this code later will be confused.